Dockerfile

```diff
@@ -4,7 +4,8 @@ MAINTAINER Michael J. Stealey <stealey@renci.org>
 ### Begin - HydroShare Development Image Additions ###
 RUN pip install --upgrade pip && pip install \
   robot_detection \
-  django-ipware
+  django-ipware \
```

Comment:
@Castronova @mjstealey Just made an edit to docker base but need permissions to create the PR